libmscore/layout.cpp

```diff
@@ -1727,14 +1727,14 @@ void Score::respace(std::vector<ChordRest*>* elements)
 
 void LayoutContext::getNextPage()
       {
-      if (!page || curPage >= score->npages()) {
-            page = new Page(score);
-            score->pages().push_back(page);
+      if (!page || curPage >= dominantScore->npages()) {
```

Comment:
Since this is based on <code>master</code>, better to start using the new coding standard?

Reply:
Done :-)